cmd/root.go

```diff
@@ -66,7 +67,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&metricsPort, "metrics-port", "p", ":8080", "The address to serve prometheus metrics.")
 	rootCmd.PersistentFlags().StringVar(&namespace, "namespace", "kube-system", "The namespace where astro is running")
 }
-func leaderElection(cmd *cobra.Command, args []string) {
+func leaderElection(_ *cobra.Command, _ []string) {
```

Comment:
was this intended?

Reply:
I can change it back but my IDE complains when there are unused vars in a function definition. We can't take them out because I think cobra requires the run function to have that function signature, so I just made them empty vars. I don't feel strongly about that though.